### PR TITLE
SUS-19: check token when removing an avatar

### PR DESCRIPTION
**Pull request by macbre**
[SUS-19 https://wikia-inc.atlassian.net/browse/SUS-19]

Check the HTTP method and edit token when trying to remove an avatar

@Wikia/sustaining-team

### CODE REVIEW

File: extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php [hunk 1]
```diff
@@ -958,6 +958,8 @@ public function renderClosingModal() {
 
 	public function removeavatar() {
```
Comment (RafLeszczynski) [inline extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php:963]: just a small thing but you could add the doc with info that this function throw BadRequest Exception

**Comment by RafLeszczynski** [comment]
one small remark, except this LGTM

File: extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php (hunk 1 above)
Comment (macbre) [inline extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php:963]: True, let's be consistent :)

0962fce

**Comment by adamkarminski** [comment]
Hetfield approves

**Pull request merged**